### PR TITLE
[ZEPL-2429] add autocompletion documentation

### PR DESCRIPTION
**Pull request by torreswoo**
# [ZEPL-2429] Autocompletion documentation

add documentation page about autocompletion
- addtional explantion about interpreter supported autocompletion.
image
image

### CODE REVIEW

**Comment by torreswoo** [comment]
i need help English explantion.

File: docs/guide/autocompletion_notebooks.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# Autocompletion in ZEPL
+
+## How to use autocompletion in ZEPL
+
+* `ctrl + .` : autocompletion shotcut
```
Comment (AhyoungRyu) [inline docs/guide/autocompletion_notebooks.md]: a typo `shotcut ` -> `shortcut ` ;)

File: docs/guide/autocompletion_notebooks.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+The Code Autocompletion can use in ZEPL notebook. When you code in Python, Spark and SQL, The Autocompletion in notebook support a better environment.
```
Comment (AhyoungRyu) [inline docs/guide/autocompletion_notebooks.md]: "can be used ?"

File: docs/guide/autocompletion_notebooks.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+<img src="../../img/autocompletion_demo.gif" class="image-box big-img" />
+
+<br>
+Don't forget that before you use autocompletion feature, the notebook should `run` at first.
```
Comment (AhyoungRyu) [inline docs/guide/autocompletion_notebooks.md]: "should be run" maybe ?

File: docs/guide/autocompletion_notebooks.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+If you find warning messsage, check to run notebook.
```
Comment (AhyoungRyu) [inline docs/guide/autocompletion_notebooks.md]: Can we be more specific in here ? for example 

`If "......" warning message shows up, check the notebook container has been started or not.`

**Review by AhyoungRyu — COMMENTED** [review]
Left some minor comments. And maybe I'm wrong too 😅
Hope @rbpark @sejunra can look through the whole content for better wording and grammar

File: docs/guide/autocompletion_notebooks.md (hunk 2 above)
Comment (sejunra) [reply docs/guide/autocompletion_notebooks.md]: Can we replace the whole sentence with:

ZEPL supports autocompletion for Python, Spark, and SQL. To begin using the autocompletion feature, simply the run the paragraph you're working on. You can run the paragraph after you select the interpreter you want to use.

File: docs/guide/autocompletion_notebooks.md (hunk 3 above)
Comment (sejunra) [reply docs/guide/autocompletion_notebooks.md]: Where is this? I don't see it...

And it should be "...before you use the autocompletion feature, first run the paragraph."

File: docs/guide/autocompletion_notebooks.md (hunk 4 above)
Comment (sejunra) [reply docs/guide/autocompletion_notebooks.md]: Can we change it to:

"If you see a warning message, check that the notebook or paragraph has run. You can do this by checking both the paragraph and the container status."

File: docs/guide/autocompletion_notebooks.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+<img src="../../img/autocompletion_error_message.png" class="image-box big-img" />
+
+<br>
+
+## Supported Interpreter Type
+
+Before starting to use it, please check autocompletion supported interpreter such as Apache Spark, Python, JDBC Interpreter.
```
Comment (sejunra) [inline docs/guide/autocompletion_notebooks.md]: Can we replace the sentence above "Before starting..." with

"Here is the list of interpreters and their specific autocompletion supported features."

**Comment by torreswoo** [comment]
@AhyoungRyu @sejunra Thanks for  your comment. i fixed it.

**Review by AhyoungRyu — APPROVED** [review]
LGTM

**Pull request merged**